src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToStartHandlerTest.java

```diff
@@ -131,7 +134,7 @@ void givenCaseHasAdjournedFieldsPopulatedAndDraftAdjournedDocs_thenDoNotClearTra
         handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
 
         Adjournment adjournment = sscsCaseData.getAdjournment();
-        assertThat(adjournment).hasNoNullFieldsOrProperties();
+        assertThat(adjournment).hasNoNullFieldsOrPropertiesExcept("interpreterLanguage");
```

Comment:
Why is this an exception here?

Reply:
because now its coming from signlnaguage service.